pre_commit_dbt/check_exposure_has_owner.py

```diff
@@ -0,0 +1,40 @@
+from pre_commit_dbt.utils import get_filenames, get_exposures
+from typing import Sequence
+from pathlib import Path
+from typing import Optional
+import argparse
+from pre_commit_dbt.utils import add_filenames_args
+
+    
+"""
+Checks if exposures have owners with a name field defined
+Arguments:
+    paths: a sequence of paths
+Returns:
+    A status code (0 if the defined exposures have owners with a defined name field and 1 if not)
+"""
```

Comment:
Is the main repository commenting their code this way? If not I think we shouldn't do it too. Can be discussed but I'm in favor of code without commentary, usually simple code with good naming is enough to understand what's going on.